src/utils/localStorage.ts

```diff
@@ -0,0 +1,3 @@
+export const localStorageTokenKey: (venueId: string | undefined) => string = (
```

Comment:
getLocalStorageTokenKey

Reply:
Given this isn't actually 'getting' anything, but just creating a string, I believe this was more correct as it's original `localStorageTokenKey` than `getLocalStorageTokenKey`